src/components/Modal/Modal.md

```diff
@@ -0,0 +1,79 @@
+## Description
+The `<Modal />` component provides a container that renders content into an accessible modal outlet. By default, the container will render `children` centered in the screen with an overlay. Out of the box, the `<Modal />` will handle setting proper focus on tabable elements within the inner content, trapping focus within the `modal`,  and toggle the state of the modal through keyboard events.
```

Comment:
what's a "modal outlet"?

Reply:
ah, the `portal` that the modal content renders in to.

Reply:
So you could just update this with:

`...into an accessible modal outlet (the portal that the modal content renders in to)`